src/cmap/connection.ts

```diff
@@ -574,6 +582,11 @@ export class Connection extends TypedEventEmitter<ConnectionEvents> {
     if (typeof options.maxAwaitTimeMS === 'number') {
       getMoreCmd.maxTimeMS = options.maxAwaitTimeMS;
     }
+    // we check for undefined specifically here to allow falsy values
+    // eslint-disable-next-line no-restricted-syntax
+    if (options.comment !== undefined) {
```

Comment:
Yes, I guess I didn't capture this in an earlier review comment. `comment` can be any BSON type, of which null is an option. We could instead change this code to be `if('comment' in options)` or some similar key existence check to permit undefined to be mapped on to the command. But by default our driver has ignoreUndefined = true so it won't actually make it to the server. If ignoreUndefined = false, undefined gets converted to a BSON `null` since BSON undefined is deprecated. Since you'd end up with null anyway in this edge case usage I think it fair for our driver to only accept/handle null. Does all that track?